test/unittests/libweb3jsonrpc/jsonrpc.cpp

```diff
@@ -729,7 +729,33 @@ BOOST_AUTO_TEST_CASE(test_importRawBlock)
     Json::Value ret;
     Json::Reader().parse(c_genesisConfigString, ret);
     rpcClient->test_setChainParams(ret);
+
     string blockHash = rpcClient->test_importRawBlock(

```

Comment:
This test actually failed to import the block (difficulty was wrong in that block after changing difficulty rules for NoProof), but `test_importRawBlock` didn't report it.

Reply:
difficulty rules are still checked for NoProof. 
nonce and mixHash are not checked.

Reply:
> difficulty rules are still checked for NoProof.
that's the reason why this test worked incorrectly (block was invalid, I guess you created it when there were no difficulty check in NoProof yet)

I changed it now to have correct difficulty.